source/agora/node/Ledger.d

```diff
@@ -108,6 +108,8 @@ public class Ledger
         // restore validator set from lastest blocks
         if (this.last_block.header.height > 0)
         {
+            UTXOSetValue[Hash] utxos =
+                this.utxo_set.getUTXOs(this.node_config.key_pair.address);
```

Comment:
```suggestion
            UTXOSetValue[Hash] utxos;
            if (this.node_config.is_validator)
                utxos = this.utxo_set.getUTXOs(this.node_config.key_pair.address);
```

Because a FullNode does not contain its own enrollments, only a Validator does.